contrib/yum/certguard.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2019 Boomi
```

Comment:
I read through this PR and it looks great. The docs all read well, and it's super useful! I hope to clean up the sphinx site and port to a read the docs site in the future, so this content will also have a nice longer term home later.

My only question is about the copyright on this line and that being confusing for our users/contributors. Having some files have different license owners and possibly different GPL versions (if we relicense to GPLv3), makes everything more complicated. Having a super clear relationship around contribution encourages contribution, and when things get complicated that maybe could stifle contribution?

I intended the copyright ownership of pulp-cerguard to be collectively owned by all contributors. Here to date there have only been two committers (mysefl and @jortel). When we merge this you'll become part of that pool of contributors. So even if this copyright and license block is removed I don't see you giving ownership or rights to another entity in any way. Nor would I want you to give that away.

What do you think about this concern? Would you be comfortable removing this block and just being considered a pulp contributor? I'm open to your thoughts and preferences here; after all it is your code.